app/views/recipients/_form.html.erb

```diff
@@ -28,7 +28,7 @@
   <div class="grid-x grid-margin-x">
     <div class="small-6 cell">
       <%= f.label :reasons_for_support %>
-      <%= f.text_area :reasons_for_support %>
+      <%= f.text_area :reasons_for_support, placeholder: "Why do you donate to this recipient?" %>
```

Comment:
We don't have placeholder texts for any of the other form text areas. To stay consistent, we could leave this one blank, too.

```suggestion
      <%= f.text_area :reasons_for_support %>
```

But maybe we should consider adding little placeholders to all or certain other fields instead.

Reply:
Yeah good call. I put it in just cause it might not be obvious what "reasons for support" means. But we should do a real pass at adding help text if we're going to go that way.